xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-theme/xwiki-platform-flamingo-theme-ui/src/main/resources/FlamingoThemes/Iceberg.xml

```diff
@@ -968,6 +968,10 @@
       <lessCode>/* Administration: Vertical Menus */
 @list-group-active-border: @list-group-border;
 
+/* Accessibility */
```

Comment:
this is not very clear why it's for accessibiility. In comments, always try to explain the WHY. Thanks

Reply:
It's also not clear why this is for accessibility and not the other color changes